docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -33,7 +33,7 @@ To deploy all the prerequisities for building Open Enclave, you can run the ```s
 
 ```powershell
 cd scripts
-.\install-windows-prereqs.ps1
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -WithFLC $true -WithAzureDCAPClient $true
```

Comment:
Suggestion, instead of -WithAzureDCAPClient perhaps something like: -WithDCAPClient <type>, where type could be: Azure, Intel, etc..?

Reply:
Well, there's only one right now :) If Intel or some other vendor produces one, then we can do this!